code/modules/holiday/holidays.dm

```diff
@@ -420,11 +384,80 @@
 				begin_day += 31
 				begin_month-- //begins in march, ends in april
 
-//	to_chat(world, "Easter calculates to be on [begin_day] of [begin_month] ([days_early] early) to [end_day] of [end_month] ([days_extra] extra) for 20[yy]")
 	return ..()
 
 /datum/holiday/easter/celebrate()
 	..()
 	GLOB.maintenance_loot += list(
 		/obj/item/weapon/reagent_containers/food/snacks/egg/loaded = 15,
 		/obj/item/weapon/storage/bag/easterbasket = 15)
+
+/datum/holiday/easter/greet()
+	return "Greetings! in the far flung past of [current_year], today would be easter, Be sure to have a happy one and keep an eye out for Bunnies!"
+
+
+/datum/holiday/easter/spess
+	name = SPACE_EASTER
+	year_offset = 540 //Canonicially the year is CURRENT YEAR + 540, so this is Easter as it would be IN SS13 itself
+
+/datum/holiday/easter/spess/greet()
+	return "Greetings! the year is [current_year+year_offset], and it's easter! be sure to have a happy one and keep an eye out for Easter Bunnies!"
+
+
+/datum/holiday/mothering_sunday
+	name = "Mothering Sunday"
+
+/datum/holiday/mothering_sunday/shouldCelebrate(dd, mm, yy)
+	if(!begin_month)
+		var/year = text2num(time2text(world.timeofday, "YYYY"))
+		var/list/motheringSundayResults = MotheringSundayDate(year)
+		end_day = begin_day = motheringSundayResults["day"]
+		end_month = begin_month = motheringSundayResults["month"]
+
+	return ..()
+
+/datum/holiday/mothering_sunday/greet()
+	return "It's Mothering Sunday! the -REAL- Mother's Day, you did remember to get her a gift right?"
```

Comment:
the -> The
But please remove the whole "the -REAL- Mother's Day, " sentence and capitalize "you".